packages/use-reapit-data/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@reapit/use-reapit-data",
+  "packageManager": "yarn@3.1.1",
+  "version": "0.0.1",
+  "private": true,
+  "description": "Data Fetching for Reapit Foundations",
+  "repository": {
+    "type": "git",
+    "url": "git+git@github.com:reapit/foundations.git"
+  },
+  "license": "MIT",
+  "author": "Reapit Ltd",
+  "main": "./src/index.ts",
+  "scripts": {
+    "start": "echo '...skipping...'",
+    "test": "cross-env TZ=UTC jest --watch --colors --silent",
+    "build": "echo '...skipping...'",
+    "lint": "eslint --cache --ext=ts,tsx,js src",
+    "check": "tsc --noEmit --skipLibCheck",
+    "release": "echo '...skipping...'",
+    "publish": "echo '...skipping...'",
+    "conf": "echo '...skipping...'",
+    "commit": "yarn test --coverage --no-cache --silent --forceExit --detectOpenHandles --runInBand --watch=false && jest-coverage-badges --input src/tests/coverage/coverage-summary.json --output src/tests/badges && yarn lint --fix && yarn check"
+  },
+  "dependencies": {
+    "@reapit/connect-session": "workspace:packages/connect-session",
+    "@reapit/foundations-ts-definitions": "workspace:packages/foundations-ts-definitions",
+    "@reapit/utils-common": "workspace:packages/utils-common",
+    "@tanstack/react-query": "^4.20.4",
+    "@tanstack/react-query-devtools": "^4.20.4",
+    "axios": "^1.2.1",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
```

Comment:
probably be better in dev deps? Or at least just `react-dom`. Our apps will likely not require react-dom for this package

Reply:
I moved just React to peer dependencies